test/ext.lean

```diff
@@ -123,3 +123,15 @@ begin
 end
 
 @[ext] structure dumb (V : Type) := (val : V)
+
+section param
+
+parameters x y : ℤ
+
+@[ext]
+lemma my_ext (s s' : set ℤ) (h : x ∈ s ↔ y ∈ s') : s = s' := sorry
```

Comment:
```suggestion
parameters x y : ℤ
variable [fact false] -- we work in an inconsistent context below

@[ext]
lemma my_ext (s s' : set ℤ) (h : x ∈ s ↔ y ∈ s') : s = s' := false.elim _inst_1
```
As of #3947 and #4025, we're now requiring tests to be silent, see https://github.com/leanprover-community/mathlib/tree/master/test#the-mathlib-test-directory.